lib/index.ts

```diff
@@ -70,8 +70,13 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
     result = { ...result, ...pluginFactory(options) };
   });
 
-  // Load bridge
-  if (bridge !== false) {
+  /*
+  ** Load bridge if you are on the Visual Editor
+  ** For more security: https://www.storyblok.com/faq/how-to-verify-the-preview-query-parameters-of-the-visual-editor
+  */
+  const isServer = typeof window === "undefined";
+  const inEditor = !isServer ? window?.location?.search?.includes('_storyblok_tk') : false;
```

Comment:
@Dawntraoz I think you could simplify this with `!isServer && window.location?.search?.includes('_storyblok_tk')`

Reply:
Done ✓